header part 1

thomsebastin opened the pull request:
Fix #44 

Test URLs:
- Before: https://main--palladium--hlxsites.hlx.page/es/cares/4-causas/infancia
- After: https://navigation--palladium--hlxsites.hlx.page/es/cares/4-causas/infancia

paolomoz commented on blocks/header/header.css:
```diff
@@ -186,6 +212,62 @@ header nav .nav-sections ul > li > ul > li {
+
+header .nav-sections > ul > li:nth-child(3) {
+  background: url("/icons/phone.svg") no-repeat;
+  background-size: cover;
+  text-indent: -1000px;
+  height: 24px;
+  width: 27px;
+  position: absolute;
+  right: 72px;
+  top: 28px;
```
I would suggest to avoid duplications and use more general selectors for the same propertu

thomsebastin replied:
I will update, Thanks.

paolomoz commented on blocks/header/header.css:
```diff
@@ -186,6 +212,62 @@ header nav .nav-sections ul > li > ul > li {
+  }
+}
+
 @media (min-width: 900px) {
```
breakpoint on the original site is 992px
we could standardize at 900 which would be better actually but we need to align between us and then with customer because the rest of the site is still on the old platform.
Sould suggest to keep 992px now. wdyt?

thomsebastin replied:
Yeah, I kinda went with the default Franklin one (600, 900 & 1200). But for now it would be a safer bet to go ahead with the ones customer has. I will update this.

paolomoz reviewed (commented):
@thomsebastin a couple of comments on top of the ones in code.

+ nav height seems to be slightly bigger than original
+ language novigation is missing (maybe this was intended in this version)
+ margin left of the right links with icons seems to be too large
+ in original navigation does not display when scrolling down

thomsebastin commented:
@paolomoz I have addressed review comments
- Removed some duplicate styles and used a different approach
- Added language icon, toggle is not done.
- Adjusted margins and media query values.

I will address the toggle and scroll logic in a separate PR.

paolomoz commented:
@thomsebastin approved, feel free to merge yourself when you like :)
